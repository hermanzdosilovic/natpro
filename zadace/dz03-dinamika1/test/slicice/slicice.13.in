60 2 7
905021
701198
709405
239589
173436
872175
951099
298153
106312
523084
121483
435014
119907
409410
959878
288435
571292
930050
707749
281048
500206
478537
512104
674329
262575
190888
382553
728453
934671
216636
335871
519826
448223
513115
902161
724120
933312
636276
724280
319600
397213
577549
860060
520117
618678
947397
307222
637770
212871
957963
518535
463015
148517
195218
898542
661290
163511
643266
731227
391104
