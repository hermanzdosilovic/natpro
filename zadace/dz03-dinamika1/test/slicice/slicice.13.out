mirko
mirko
mirko
slavko
slavko
slavko
slavko
mirko
mirko
mirko
mirko
mirko
slavko
slavko
mirko
slavko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
mirko
slavko
mirko
mirko
mirko
slavko
slavko
slavko
mirko
mirko
mirko
mirko
mirko
slavko
slavko
mirko
mirko
mirko
mirko
mirko
mirko
slavko
slavko
mirko
slavko
slavko
slavko
slavko
mirko
mirko
mirko
slavko
slavko
mirko
slavko
mirko
slavko
